ocs_ci/framework/pytest_customization/ocscilib.py

```diff
@@ -137,8 +137,8 @@ def pytest_configure(config):
             # add csi versions
             csi_versions = get_csi_versions()
             config._metadata['csi-provisioner'] = csi_versions.get('csi-provisioner')
```

Comment:
csi-provisioner was also None I saw, but do not see this fix is touching this one?

Reply:
I didn't see any reason why csi-provisioner is None in email where this problem is initiated. I have tested with bot upstream and downstream builds with this PR and its able to get the version as expected.